Marlin/SanityCheck.h

```diff
@@ -369,5 +369,8 @@
 #elif defined(SDEXTRASLOW)
   #error SDEXTRASLOW deprecated - set SPI_SPEED to SPI_QUARTER_SPEED instead
 #endif
+#ifdef Z_RAISE_BEFORE_HOMING
```

Comment:
Might.
The work is done by the compiler.
All this errors are completely independent from each other.
If no error is detected all inditions have to be evaluated anyway.
If any error is detected the compilation process stops anyway.

So it does not make a difference between #if and #elif - just an optical one.